app/js/components/LabOrderListFIlters.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {
+  FormControl,
+} from 'react-bootstrap';
+import CustomDatePicker from './shared/CustomDatePicker';
+import Dropdown from './shared/Dropdown';
+
+class LabOrderListFilters extends PureComponent {
+  renderNameEMROrOrderIdFilter() {
+    const { handleFieldChange, clearNameEMRField, nameField } = this.props;
+    return (
+      <span className="form-filter-group one-third">
+        <span className="filter-label">Search for a sample</span>
+        <span className="name-emrid-order-filter">
+          <i className="small icon-search" />
+          <FormControl
+            type="text"
+            placeholder="search by ID or name, or scan sample"
+            value={nameField}
+            onChange={event => handleFieldChange('nameField', event.target.value)}
+          />
+          <i className="small scale icon-remove-sign" role="toolbar" onClick={event => clearNameEMRField()} />
+        </span>
+      </span>
+    );
+  }
+
+  renderDatePickerFilters() {
+    const { handleFieldChange } = this.props;
+    return (
+      <span className="date-picker-filter">
+        <span>
+          <CustomDatePicker
+            labelClassName="filter-label line"
+            label="From: "
+            defaultDate={moment().subtract(7, 'days')}
```

Comment:
I see you are using moment... that is what we always used to use, though I though there was a recent preference to use date-fns instead?  I don't have a strong preference either way, but let's make sure we just are consistent and use on or the other in this module.